openapi.yaml

```diff
@@ -471,6 +591,173 @@ paths:
           $ref: "#/components/responses/InternalServerError"
 components:
   schemas:
+    AddRelationArgs:
+      type: object
+      properties:
+        endpoints:
+          type: array
+          items:
+            type: string
+        via-cidrs:
+          type: array
+          items:
+            type: string
+      additionalProperties: false
+      required:
+        - endpoints
+    Application:
+      type: object
+      description: Information about a given application.
+      properties:
+        base:
+          $ref: '#/components/schemas/Base'
+        channel:
+          type: string
+        charm:
+          type: string
+        constraints:
+          $ref: '#/components/schemas/Constraints'
+        endpoint-bindings:
+          type: object
+          patternProperties:
+            .*:
+              type: string
+        exposed:
+          type: boolean
+        exposed-endpoints:
+          type: object
+          patternProperties:
+            .*:
+              $ref: '#/components/schemas/ExposedEndpoint'
+        life:
+          type: string
+        principal:
+          type: boolean
+        remote:
+          type: boolean
```

Comment:
What is this?

Reply:
That's a flag indicating if this application is a RemoteApplication involved in a cross-model relation. I always find this flag weird, because the application is "remote" only from the other side's point of view. We can take it out for now and bring it back when thinking about cross-model relations. WDYT?